src/content/guides/pay-payment-app-integration.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Paying a Payment Request
+description: Integrate your app with Centrapay to pay payment requests on merchant terminals and POS
+nav:
+  path: Reference/App Integrations
+  order: 1
+---
+
+## Overview
+
+This guide is for a [Third-party Asset Provider](/guides/integrating-third-party-asset) to enable paying with their
+digital assets via Centrapay.
+
+## Authenticating API calls on behalf of users
+
+Third-party applications can make API calls on behalf of users with Centrapay
+API keys. All API requests to Centrapay must be made from the integrator’s backend.
+Centrapay API keys must also be managed on the backend and stored as secrets to
+minimize the risk of API keys being compromised.
+
+Seperate API keys will be used for test and live payments.
+
+> Contact [integrations@centrapay.com](mailto:integrations@centrapay.com) to be issued your API keys.
+
+## Payment Flow Implementation
+
+The payment flow below describes how a Third-party Asset Provider should integrate
+with the [Dynamic Merchant QR Code flow](/guides/payment-flows#dynamic-merchant-qr-code) to enable digital payments within their
+application. Below is an overview of the process, followed by a more in-depth
+look at the integrator’s responsibilities.
+
+> Note this is assuming the best case scenario, and not accounting for edge cases.<br/><br/>
```

Comment:
There should be a way to make a multi-line blockquote without adding HTML to our Markdown

Reply:
I'm not sure if this note is providing much value - if we follow this pattern, we'd need to put it on every diagram

Reply:
I agree I think that it is inferred as part of good design

Reply:
@tom-blake I think it's valuable to add for now. I think in the future we should have an intro guide that would contain the common principles

Reply:
@IsaacMurtagh Perhaps this is better being expanded on in the testing section instead of introducing noise here? All this callout does is raise more questions.

I am concerned that referring to "edge cases" without documenting them makes our solution look flaky. It's also technical jargon. I'd prefer we remove this line completely and provide useful testing steps for integrators so that they're properly informed

Reply:
I've removed this section. Think the diagram can be inferred to be happy path without extra context.